src/utils/v2/fundingCycle.ts

```diff
@@ -194,3 +194,22 @@ export const V2FundingCycleRiskCount = (
     v => v === true,
   ).length
 }
+
+export const deepEqV2FundingCycles = (
```

Comment:
whats dis for?

Reply:
Oh yeah shit accidentally leaked some funding history tab shit into this branch. Thought I had removed it all but I missed this, cheers